jpa/eclipselink.jpa.test/src/it/java/org/eclipse/persistence/testing/tests/jpa22/advanced/StoredProcedureQueryTestSuite.java

```diff
@@ -1056,6 +1057,33 @@ public void testStoredProcedureQuerySysCursor2() {
         }
     }
 
+    /**
+     * Tests a StoredProcedureQuery using a system cursor. Also tests
+     * getParameters call AFTER query execution. Parameters are passed via position.
+     */
+    public void testStoredProcedureQuerySysCursor2_Positional() {
+        if (supportsStoredProcedures() && getPlatform().isOracle() ) {
```

Comment:
is `supportsStoredProcedures()` not enough? What about running this on `getPlatform().isMySQL()`?